packages/flutter_test/lib/src/goldens.dart

```diff
@@ -314,6 +342,21 @@ class _TrivialWebGoldenComparator implements WebGoldenComparator {
   Uri getTestUri(Uri key, int? version) {
     return key;
   }
+
+  @override
+  Future<bool> compareBytes(Uint8List bytes, Uri golden) {
+    // Ideally we would use markTestSkipped here but in some situations,
+    // comparators are called outside of tests.
+    // See also: https://github.com/flutter/flutter/issues/91285
+    // ignore: avoid_print
+    print('Golden comparison requested for "$golden"; skipping...');
+    return Future<bool>.value(true);
```

Comment:
Instead of duplicating the logic, consider moving it into a common private method and invoking that method from both `compare` and `compareBytes`.

Reply:
Done